src/Tasks.UnitTests/AssemblyDependency/Miscellaneous.cs

```diff
@@ -3312,7 +3312,7 @@ public void ParentAssemblyResolvedFromAForGac()
         /// <returns></returns>
         private ReferenceTable GenerateTableWithAssemblyFromTheGlobalLocation(string location)
         {
-            ReferenceTable referenceTable = new ReferenceTable(null, false, false, false, false, new string[0], null, null, null, null, null, null, SystemProcessorArchitecture.None, fileExists, null, null, null, null,
+            ReferenceTable referenceTable = new ReferenceTable(null, false, false, false, false, new string[0], null, null, null, null, null, null, SystemProcessorArchitecture.None, fileExists, getDirectoryFile, null, null, null, null,
```

Comment:
not actionable....I am unsure how I feel about this.  I know you only updated a parameter.

Reply:
Refactoring unit tests, although it would be beneficial, was not target of this PR.
Possible action would be to create 'issue' to do so - if there is reasonable chance anyone will get to it.